.github/workflows/ci.yml

```diff
@@ -56,9 +56,13 @@ jobs:
         if: matrix.features == ''
         run: cargo build --all-features
 
-      - name: Test all features
-        if: matrix.features == ''
-        run: cargo test --all-features
+      - name: Test all features (macOS)
+        if: matrix.features == '' && runner.os == 'macOS'
+        run: cargo test --all-features -- --skip set_deadline_policy
+
+      - name: Test all features (other)
+        if: matrix.features == '' && runner.os != 'macOS'
+        run: sudo -E $HOME/bin/cargo test --all-features
```

Comment:
Out of curiosity, why was this necessary?

Reply:
macOS doesn't implement `SCHED_DEADLINE` since it's not part of POSIX. The test will only pass on Linux. It looks like the rest of the tests are failing due to cargo being installed in a different location than I expected, and `sudo` not being part of the environment. The test on Linux needs to be executed as root.

Reply:
I'm not familiar enough with GitHub Actions to know what the correct fix is here. I assumed that the run command would be executed as a regular shell command inside a VM, but it appears that was wrong.

Reply:
Can you try changing it to `sudo -E /usr/share/rust/.cargo/bin/cargo test --all-features` please?

Reply:
Try again now.

Reply:
Looks like that (mostly) worked, I pushed up a typo fix. Sorry for the spam, I'm not super familiar with GitHub Actions.